### DISCUSSION TITLE
Video start delay

### DISCUSSION THREAD

**Discussion by jocelynetienne** [pdfpc/pdfpc#731] [Q&A] [answered]
I have a long-ish delay (from 5 up to 30 sec) when I press play on a video. It is not associated with any process being high on CPU usage. The video then plays at its normal rate.

This is debian with stable `pdfpc` 4.6 and gstreamer1.0-gtk3 Version: 1.22.0-5+deb12u1.

(Also, is there an option for pdfpc to honour the `autostart` flag in the multimedia package of beamer? Okular does)

**Comment by fnevgeny** [comment, marked as answer]
> I have a long-ish delay (from 5 up to 30 sec) when I press play on a video. It is not associated with any process being high on CPU usage. The video then plays at its normal rate.

If you believe it's a pdfpc bug, please report it [as such](https://github.com/pdfpc/pdfpc/issues/new?assignees=&labels=&projects=&template=Bug_report.md).

> (Also, is there an option for pdfpc to honour the `autostart` flag in the multimedia package of beamer? Okular does)

Have a look at the [demo](https://github.com/pdfpc/pdfpc/releases/latest/download/pdfpc-video-example.zip).
